themes/bootstrap3/templates/search/facet-list.phtml

```diff
@@ -11,6 +11,7 @@
     $searchAction .= urlencode($this->baseUriExtra);
     $urlBase .= '&amp;baseUriExtra=' . urlencode($this->baseUriExtra);
   }
+  $this->headTitle($this->translate('Facet List for ' . $this->facet));
```

Comment:
It's not a good idea to create translation strings that rely on concatenation, because some languages use different word orders. It would be better to use a token, like `$this->translate('Facet List for %%field%%', ['%%field%%' => $this->facet])`. If we do that, we also need to add the new string to the language files.

Reply:
I've fixed this, for English. It's not clear to me if there's an easy way to add things like this to the other translations using some sort of automated translating.

Reply:
@Wooble, that's fine. There is no automated translating, but as long as new strings are added to the English file, I contact translators and ask for missing translations every time we plan a new release, so this will be included in that process.